README.md

```diff
@@ -59,9 +59,13 @@ Per-VDOM:
    * `fortigate_interface_receive_bytes_total`
    * `fortigate_interface_transmit_errors_total`
    * `fortigate_interface_receive_errors_total`
- * _VPN/Ssl_
+ * _VPN/Ssl/Connections_
    * `fortigate_vpn_connections`
    * `fortigate_vpn_users`
+ * _VPN/Ssl/Stats_
+   * `fortigate_vpn_ssl_current_users`
+   * `fortigate_vpn_ssl_current_tunnels`
+   * `fortigate_vpn_ssl_current_connections`
```

Comment:
The `current` part is implied, and the other VPN metric does not have it, so we should drop it.

```suggestion
   * `fortigate_vpn_ssl_users`
   * `fortigate_vpn_ssl_tunnels`
   * `fortigate_vpn_ssl_connections`
```

Reply:
Ok, this was just to differenciate the 2 sections of the API call
```
  {
    "http_method":"GET",
    "results":{
      "conserve_mode":false,
      "max":{
        "users":7,
        "tunnels":6,
        "connections":12
      },
      "current":{
        "users":3,
        "tunnels":2,
        "connections":2
      }
    },
```
Ok for dropping it